README.md

```diff
@@ -2,7 +2,20 @@
 <h3 align="center"> Noise Supression Application for PulseAudio or Pipewire</h3>
 <p align="center"><img src="https://raw.githubusercontent.com/noisetorch/NoiseTorch/master/assets/icon/noisetorch.png" width="100" height="100"></p> 
 
-[![License: GPL v3](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://www.gnu.org/licenses/gpl-3.0) [![Last Release](https://img.shields.io/github/v/release/noisetorch/NoiseTorch?label=latest&style=flat-square)](https://github.com/noisetorch/NoiseTorch/releases)
+
+<div align="center">
+    
+  <a href="">[![Licence][licence]][licence-url]</a>
+  <a href="">[![Latest][version]][version-url]</a>
+    
+</div>
+
+[licence]: https://img.shields.io/badge/License-GPLv3-blue.svg
+[licence-url]: https://www.gnu.org/licenses/gpl-3.0
+[version]: https://img.shields.io/github/v/release/noisetorch/NoiseTorch?label=Latest&style=flat-square
```

Comment:
```suggestion
[version]: https://img.shields.io/github/v/release/noisetorch/NoiseTorch?label=Latest&style=flat
```

Just noticed this, the GPL badge has rounded corners, it may look better if we have both badges in the same style.